h/static/styles/partials-v2/_tooltip.scss

```diff
@@ -0,0 +1,74 @@
+@mixin tooltip-arrow($rotation) {
+  background: $grey-7;
+  border-bottom: 1px solid rgba(0,0,0,0.20);
+  border-right: 1px solid rgba(0,0,0,0.20);
+  content: "";
+  display: none;
+  height: 6px;
+  left: 0;
+  margin-left: auto;
+  margin-right: 7px;
+  position: absolute;
+  right: 0;
+  transform: rotate($rotation);
+  width: 6px;
+
+  .env-js-capable & {
+    display: block;
+  }
+
+  .env-js-timeout & {
+    display: none;
+  }
+}
+
+.tooltip {
+  @include font-small;
+
+  border-radius: 2px;
+  background-color: $grey-7;
+  color: white;
+  display: none;
+  font-style: normal;
+  padding-left: 5px;
+  padding-right: 5px;
+  padding-top: 4px;
+  padding-bottom: 4px;
+  position: fixed;
+  max-width: 220px;
+  z-index: $zindex-tooltip;
+
+  .env-js-capable & {
+    display: block;
+  }
+
+  .env-js-timeout & {
+    display: none;
+  }
+}
+
+// Arrow at the bottom of the tooltip pointing down at the target element.
+.tooltip:before {
+  @include tooltip-arrow(45deg);
+  content: "";
+  top: calc(100% - 5px);
+  right: calc(100% - 17px);
+}
+
+.tooltip-label {
+  // Make the label a positioned element so that it appears _above_ the
+  // tooltip's arrow, which partially overlaps the content of the tooltip.
+  position: relative;
+}
+
+@include touch-input {
```

Comment:
I thought the intention was to display the label on touch devices rather than rely on the hint icon? When viewing the page with http://localhost:5000/account/profile?__env__=touch I still see the tooltip icon rather than the hint text below the field.